lib/PnpPlugin.js

```diff
@@ -77,10 +79,18 @@ module.exports = class PnpPlugin {
 						return;
 					}
 
+					if (typeof resolution === "string") {
+						resolution = transformPathToPosix(resolution);
```

Comment:
Same here

Reply:
external path, that can be received from pnpapi -> we transform it to path with posix separators in case we get win32 separator from outside